src/cart/crt_init.c

```diff
@@ -273,6 +275,21 @@ static int data_init(int server, crt_init_options_t *opt)
 	D_DEBUG(DB_ALL, "Starting RPCID %#lx. Num cores: %ld\n",
 		crt_gdata.cg_rpcid, crt_gdata.cg_num_cores);
 
+	/* Set context post init / post incr to tune number of pre-posted recvs */
+	if (getenv("D_POST_INIT") != NULL) {
+		uint32_t post_init = 0;
+		d_getenv_int("D_POST_INIT", &post_init);
+		crt_gdata.cg_post_init = post_init;
+	} else
+		crt_gdata.cg_post_init = CRT_HG_POST_INIT;
+
+	if (getenv("D_POST_INCR") != NULL) {
+		uint32_t post_incr = 0;
+		d_getenv_int("D_POST_INCR", &post_incr);
+		crt_gdata.cg_post_incr = post_incr;
+	} else
+		crt_gdata.cg_post_incr = CRT_HG_POST_INCR;
+
```

Comment:
[Not requesting changes] I think `d_getenv_int` is designed to be used in the following manner:

```
uint32_t post_init = CRT_HG_POST_INIT;

d_getenv_int("D_POST_INIT", &post_init):
```

Reply:
oh yes you're right I should do that instead, thanks I will push the change.

Reply:
Thanks.